modules/candidate_parameters/jsx/monitoring.js

```diff
@@ -0,0 +1,449 @@
+              onUserInput ={this.setFormData}
+              ref ={monitored}
+              disabled ={disabledFields[i] || disabled}
+              required ={monitoredRequired[i]}
+            />
+          );
+          visits.push(
+            <DateElement
+              label ={dateMonitoredLabel}
+              name ={dateMonitored}
+              value ={this.state.formData[dateMonitored]}
+              onUserInput ={this.setFormData}
+              ref ={dateMonitored}
+              disabled ={disabledFields[i] || disabled}
+              required ={elRequired[i]}
+            />
+          );
+          visits.push(
+            <DateElement
+              label ={dateMonitored2Label}
+              name ={dateMonitored2}
+              value ={this.state.formData[dateMonitored2]}
+              onUserInput ={this.setFormData}
+              ref ={dateMonitored2}
+              disabled ={disabledFields[i] || disabled}
+              required ={elRequired[i]}
+            />
+          );
+          visits.push(
+            <SelectElement
+              label ={monitorIDLabel}
+              name ={monitorID}
+              options ={this.state.Data.examiners}
+              value ={this.state.formData[monitorID]}
+              onUserInput ={this.setFormData}
+              ref ={monitorID}
+              disabled ={disabledFields[i] || disabled}
+              required ={elRequired[i]}
+            />
+          );
+
+          visits.push(<hr />);
+          i++;
+        }
+      }
+
+      var formattedHistory = [];
+      for (let row in this.state.Data.history) {
+        if (this.state.Data.history.hasOwnProperty(row)) {
+          let line = [];
+          let lineInfo = "";
+          let historyRow = this.state.Data.history[row];
+          let userH = historyRow.entry_staff;
+          let updateDateH = historyRow.data_entry_date;
+          let visitH = historyRow.visit_label;
+          let flaggedH = historyRow.flag;
+          let monitoredH = historyRow.monitored;
+          let monitoredDateH = historyRow.date_monitored;
+          let examinerH = historyRow.monitor_id;
+
+          lineInfo += "[UPDATE by ";
+          lineInfo += userH;
+          lineInfo += " at ";
+          lineInfo += updateDateH;
+          lineInfo += "]";
+
+          line.push(
+            <td>
+              <b>
+                Visit->
+              </b>
+              {visitH}
+            </td>
+          );
+          line.push(
+            <td class="td-hist">
+              <b>
+                Flagged ->
+              </b>
+              {flaggedH}
+            </td>
+          );
+          line.push(
+            <td class="td-hist">
+              <b>
+                Monitored ->
+              </b>
+              {monitoredH}
+            </td>
+          );
+          if (monitoredDateH !== null) {
+            line.push(
+              <td class="td-hist">
+                <b>
+                  Date Monitored ->
```

Comment:
i'm a little confused by this. does it mean "Date monitoring was completed"? i was under the impression that data monitoring is not necessarily contained within a single day; but i'm not sure if asking for a data monitoring start date would be appropriate or useful either. at the very least, this could also be slightly more descriptive imo